src/directions/directions_service.ts

```diff
@@ -271,11 +271,11 @@ export class MigrationDirectionsService {
       const googleRoute: google.maps.DirectionsRoute = {
         bounds: bounds,
         legs: googleLegs,
+        copyrights: AWS_COPYRIGHT,
+        summary: route.MajorRoadLabels?.[0]?.RoadName?.Value || "",
```

Comment:
If there are multiple major road labels I believe they get combined, try this on a long route such as Austin -> Chicago